lib/block.rb

```diff
@@ -0,0 +1,24 @@
+require 'pry'
+require 'date'
+require_relative 'booking'
+
+module Hotel
+  class Block < Booking
+    attr_reader :id, :rooms, :date_range, :nights, :total_cost, :block_info, :block_name, :discount, :reserved_rooms
+
+    def initialize(id, rooms, date_range, block_info)
+      super(id, rooms, date_range)
+      @block_info = block_info
+    end
+
+    def rooms_available_block
```

Comment:
Because this method is already inside the `Block` class, you shouldn't need to add `_block` to the name.